docs: form tour new fields (#228)
frappe/frappe_docs: frappe_docs/www/docs/user/en/form-tours.md

To create a Form Tour, type "new form tour" in awesomebar and hit enter.

1. Enter Title. For eg., 'Creating a Custom Field'
1. Select Reference DocType.
1. Add steps defining each fields.
1. Save the document.

![Custom Field Form Tour](/docs/assets/img/custom-field-form-tour.png)
*A Tour to explain creation of custom fields*

## Configuration Options

### Form Tour

1. **Is Standard**: To make a standard Form Tour which will be stored as JSON. Can only be set while developer mode is on.
1. **Save on Completion**: If checked, the last step of the Form Tour will prompt the user to save the document.
1. **Show First Document Tour**: If you want to show the tour of an existing document instead of a new form, enable this. As the name suggests, it gives the tour of the very first document created for this DocType.

> 1. **Include Name Field**: Some DocTypes have user set name fields. On enabling this, the name field becomes the first step of the tour.

Reviewer: ```suggestion
1. **Include Name Field**: For some doctypes, the name is set by the user. On enabling this, the name field becomes the first step of the tour.
```
